mozetl/taar/taar_similarity.py

```diff
@@ -26,6 +26,9 @@
 CATEGORICAL_FEATURES = ["geo_city", "locale", "os"]
 CONTINUOUS_FEATURES = \
     ["subsession_length", "bookmark_count", "tab_open_count", "total_uri", "unique_tlds"]
+# Set a constrain on the pairwise comparisons that are allowed
+#  both at an intra- and inter-custer level to avoid OOM errors in KDE.
+KDE_SIZE_THRESH = 1000
```

Comment:
How was this picked? Did you check that this constant helps fixing the OOM situation?